### ISSUE TITLE
i18n/l10n (Translation / Localization)

### ISSUE THREAD

**Issue by Drachenkaetzchen** [partkeepr/PartKeepr#27]
We want the system to speak your language. Technically, all JS code is wrapped by i18n() which does the translation later - we just need to implement it and create a gettext parser to eat JS files.
##

**Comment by MarPiRK** [comment]
I'm interested in translating to Polish. Is it possible in current version?

**Comment by felicitus** [comment]
No, it's not possible yet.

**Comment by DIdro** [comment]
I'm also interesting in translation.
I see, that Symphony2 have some localization and internationalization features. Is it possible to use it in porject?

Thanks

**Comment by felicitus** [comment]
@DIdro yes and no - the ExtJS translation component is missing and needs to be developed.

**Comment by felicitus** [comment]
Extraction can be implemented using this guide: http://jmsyst.com/bundles/JMSTranslationBundle/master/usage#extracting-translation-messages

Open Issue: How to pass the translation to the frontend?

**Comment by iled** [comment]
Hi,
Any update on how to contribute to i18n?

**Comment by felicitus** [comment]
i18n is not implemented yet, thus there's no way to contribute to i18n (if you mean translations).

**Comment by caladanbr00d** [comment]
Are there any plans on integrating translation features?
